backend/repositories/generalRepository.js

```diff
@@ -45,6 +45,8 @@ function setObjPropsById(id, setObj, callback) {
         _id: id
     }, {
         $set: setObj
+    }, {
+        multi: false
```

Comment:
We make query by Id. It would always one record, so in this case this flag isn't necessary